### ISSUE TITLE
Remove Java Web Start references from the code...

### ISSUE THREAD

**Issue by dmatej** [eclipse-ee4j/glassfish#24056]
1. Deprecated in JDK9, killed in JDK11
2. JNLP Templates in GlassFish did not work 12 years ago (the log manager really doesn't exist so long): https://download.oracle.com/javaee-archive/glassfish.java.net/users/2010/02/42161.html

**Comment by dmatej** [comment]
Another possibility is to fix it and write at least one test for it. What do you think, is it worth of it?

**Comment by jaceabreu** [comment]
I suggest just getting rid of it altogether.  Wasn't it deprecated in JDK 9, to be removed soon, anyways?

**Comment by PatrickHuang888** [comment]
agree, should clean this code when refactoring. and maybe this will involve TCK test case refactoring too.

**Comment by renatsaf** [comment]
Hi @OndroMih. I asked Claude to analyze the issue.

**Comment by renatsaf** [comment]
**Recommendation: remove, not fix.** Java Web Start was removed from the JDK in 11, and GlassFish 7.x targets JDK 11+, so this code path cannot run on any supported JVM — there's no value in fixing/testing it. The `java-web-start-access` element is a GlassFish-proprietary *runtime* DD extension (it isn't part of the standard `application-client` DD), so the official Jakarta EE TCK doesn't assert it. The only test fallout is GlassFish's own `appserv-tests` devtests, which we can edit freely. @PatrickHuang888 — I think that addresses the TCK worry; the binding tests are all in-repo.

I'd suggest splitting the removal into 3 independently buildable/revertible PRs, ordered so behavior is removed first and the rest is pure dead-reference cleanup:

**PR 1 — delete the JWS engine** (no public API, safe to land first)
- `appclient/server/core/.../jws/` — the full package (~24 classes: `JavaWebStartInfo`, `JWSAdapterManager`, `AppClientHTTPAdapter`, `DeveloperContentService`, `servedcontent/`, `templates/*.jnlp`, …) and its resources (`.jnlp` templates + `LocalStrings`)
- `appclient/client/acc/.../jws/boot/` — `ClassPathManager{,15,16}`, `JWSACCMain`, `ErrorDisplayDialog`, `jwsclient.policy`
- Fix up referrers: `CarHandler` (uses `JavaWebStartInfo`'s logger constants — relocate them), `AppClientDeployer` (the config-change listener), `NestedAppClientDeployerHelper` (`library.jnlp.path`)

**PR 2 — DOL descriptor + DTDs**
- Delete `JavaWebStartAccessNode`, `JavaWebStartAccessDescriptor`, `JnlpDocNode`, `JnlpDocDescriptor`
- Drop the `registerElementHandler(... JAVA_WEB_START_ACCESS ...)` line and `writeJavaWebStartInfo()` call in `AppClientRuntimeNode`, plus `JAVA_WEB_START_ACCESS` in `RuntimeTagNames`
- Remove `java-web-start-access?` / `jnlp-doc` from the app-client DTDs (`sun-application-client_5_0-0`, `_6_0-0`, `glassfish-application-client_6_0-1/-2`)
- Update the one active devtest DD (`devtests/ejb/mdb/singleton/descriptor/glassfish-application-client.xml`) and the `v2-tests` appclient fixtures

**PR 3 — `java-web-start-enabled` config attribute + admin/GUI/docs**
- `JAVA_WEB_START_ENABLED` in `ServerTags`, the `AppclientModule` config bean accessor, `setJavaWebStartEnabled` in `DeploymentProperties`/`DFDeploymentProperties`, and the legacy migration in `UpgradeService`
- AdminGUI: the (already-disabled) checkbox in `edit.inc`, `deploymentAppClient.jsf`, `redeployFrame.jsf`, related `showJavaWebStart`/`convertToFalseProp` flags, `DeploymentHandler`, `ApplicationHandlers`
- Docs/manpages: `java-clients.adoc` (ch. 11), `deploy`/`redeploy`/`deploydir` `.adoc`+`.1`, `dd-elements.adoc`/`dd-files.adoc`, and the `.jnlp` MIME mapping in `default-web.xml`

Each PR builds and reverts on its own; PR 1 removes all behavior, so PRs 2–3 are just cleanup of now-dead references. Happy to take the first PR if there's agreement on the approach.

**Comment by dmatej** [comment]
@renatsaf What about https://github.com/karakun/OpenWebStart + https://openwebstart.com/ ?

And I will answer myself why I see removal as the better option: looking at the GitHub repository and root POM, it still uses Java 1.8. So I see it basically in some conserved maintenance mode, no future ahead.

**Comment by renatsaf** [comment]
> [@renatsaf](https://github.com/renatsaf) What about https://github.com/karakun/OpenWebStart ?
I'l ask Claude to analyze pros/contras.

**Comment by dmatej** [comment]
However I would prefer "atomic" removal, so one PR, not three. And the work should be done in several commits (not PRs).

**Comment by renatsaf** [comment]
Good point — OWS does keep JNLP alive (v1.13.0 last month, runs on JDK 8–21), so I'll walk back "no supported JVM can run it." The format isn't dead; the question is just GlassFish's *generator*, which is broken 12+ years per your own note above, and whose client ACC bootstrap (`ClassPathManager15/16`) reflects into classloader internals that strong encapsulation on JDK 17/21 would likely break. One thing that's true either way: there's no license concern — GlassFish only *emits* JNLP + signed JARs and the user installs OWS separately, so OWS's GPLv2 never touches our tree.

So the choice isn't really forced by "can it run" anymore; it comes down to ownership and appetite. Roughly three paths:
- **A — Remove** the JWS generator, keep the plain `appclient` command-line launcher (which isn't JWS and works regardless).
- **B — Revive** it targeting OWS: fix the generator, add an OWS-based smoke test, and maintain it going forward.
- **C — Remove now, reintroduce later** as a fresh, tested OWS generator if/when a contributor wants it, rather than keeping the rotted code in-tree.

Happy to go whichever way the team prefers — let me know if there's a sense of how many users still rely on the app-client launch path, since that probably decides between A and B.

**Comment by renatsaf** [comment]
Hi @dmatej what do you think?
